index.bs

```diff
@@ -1918,12 +1963,26 @@ The [=remote end steps=] with |session| and |command parameters| are:
 
       1. [=set/Append=] |navigable|'s [=navigable id=] to |top-level traversable context ids|.
 
+1. Otherwise, if |input user context ids| is not empty:
```

Comment:
Aren't we missing the definition of `|input user context ids|`?

Also I think we need to error if both |input user context ids| and |input context ids| are set (and should be clear in the definition of a subscription that even though it's a product type, those fields are mutually exclusive for any given subscription).

Reply:
Indeed, let me check, I think I may have lost it while rebasing